testing/test_pytest_html.py

```diff
@@ -305,7 +305,12 @@ def pytest_runtest_makereport(item, call):
             content)
         assert link in html
 
-    def test_extra_image(self, testdir):
+    @pytest.mark.parametrize('mime_type, extension',
+                             [('image/png', 'png'),
+                              ('image/png', 'image'),
+                              ('image/jpeg', 'jpg'),
+                              ('image/svg+xml', 'svg')])
+    def test_extra_image(self, testdir, mime_type, extension):
```

Comment:
I did not add a `test_extra` yet. I thought the test would be cleaner. Because I have to put two extra `if` in the test.
```
if file_type == 'json:
  content_str = json.dumps(content)
```
If you think this is not a problem, I can merge the test in one.